clients/deltalake/src/test/java/org/projectnessie/deltalake/ITDeltaLogBranches.java

```diff
@@ -45,7 +47,7 @@
 
 class ITDeltaLogBranches extends AbstractSparkTest {
 
-  private NessieClient client;
+  static NessieApiV1 api;
```

Comment:
nit: why `static`?

Reply:
BTW: that's code from #1932 (which is why this PR is in draft)